src/pages/images/actions/UploadCustomIsoBtn.tsx

```diff
@@ -20,7 +21,8 @@ const UploadCustomIsoBtn: FC<Props> = ({ className }) => {
   const handleFinish = (name: string) => {
     toastNotify.success(
       <>
-        Image <b>{name}</b> uploaded successfully
+        Image <ResourceLabel type="iso-volume" value={name} /> uploaded
```

Comment:
```suggestion
        Custom ISO <ResourceLabel type="iso-volume" value={name} /> uploaded
```
I think this was a bug? Maybe check the rest of the file, it might be wrong in errors as well.